Dockerfile

```diff
@@ -1,35 +1,31 @@
 # Perform common operations, dependency installation etc...
-FROM python:3.9.0-buster as base
+FROM python:3.8-slim-buster as base
 RUN pip install poetry
+RUN pip install gunicorn
+RUN pip install flask
```

Comment:
Note for myself that we discussed this on the support call - docker-ignoring the `project.toml` file may remove the need for explicitly installing these, but I'm not too worried if you're using this workaround here